course_size_lx: 5.22
course_size_ly: 3.96
course_size_r : 3.3
course_size_lx_out: 6.75
course_size_ly_out: 8.5


